assets/monitoring-plugin/console-plugin.yaml

```diff
@@ -16,3 +16,5 @@ spec:
       port: 9443
     type: Service
   displayName: monitoring-plugin
+  i18n:
```

Comment:
these are generated via jsonnet, you'd need to add that here https://github.com/openshift/cluster-monitoring-operator/blob/df5e2abc418929264edc59193bc3641b811d461b/jsonnet/components/monitoring-plugin.libsonnet#L75, then `make build-jsonnet`

Thanks for this.